docs/src/Components.jsx

```diff
@@ -81,6 +81,29 @@ var Components = React.createClass({
           </ul>
         </article>
 
+        <article>
+          <h3 id="credit_card_number">Credit Card Number</h3>
+          <p>Masked Credit Card Number input</p>
```

Comment:
The number isn't "Masked" as I expected... is this a correct description? I would have expected to see it mask as I type (like a password field) based on this description.